parachain/README.md

```diff
@@ -153,6 +153,25 @@ For interacting with our chain using the Polkadot-JS API, you'll need to supply
   "BridgedAssetId": "H160",
   "AssetAccountData": {
     "free": "U256"
+  },
+  "EthHeader": {
+    "parentHash": "H256",
+    "timestamp": "u64",
+    "number": "u64",
+    "author": "H160",
+    "transactionsRoot": "H256",
+    "ommersHash": "H256",
+    "extraData": "Vec<u8>",
+    "stateRoot": "H256",
+    "receiptsRoot": "H256",
+    "logBloom": "Bloom",
+    "gasUsed": "U256",
+    "gasLimit": "U256",
+    "difficulty": "U256",
+    "seal": "Vec<Vec<u8>>"
+  },
```

Comment:
This was intended to be in my next PR.... whoops... I think it's harmless here though